en/summary.rst

```diff
@@ -109,3 +110,4 @@ Hosting guide
 .. _`Use a specific backbone view`: /en/developer_guide/specific_backbone_view.rst
 .. _`Add a special listener to a Content field`: /en/developer_guide/content_add_field_listener.rst
 .. _`Use the ajax pagination for the entities list`: /en/developer_guide/entity_list_ajax_pagination.rst
+.. _`Creating a new site` : /en/hosting_guide/site_creation.rst
```

Comment:
Could you respect the tree :) 
(we put the new line between line having the same size )